xyniceboy commented on Mar 10, 2016
operation
import commands
commands.getstatusoutput('scrapyd-deploy crawlers -p crawlers_hoscs')
result
Deploying to project "crawlers_hoscs" in http://192.162.25.1:6800/addversion.json
Traceback (most recent call last):
File "/usr/local/bin/scrapyd-deploy", line 273, in
main()
File "/usr/local/bin/scrapyd-deploy", line 96, in main
if not _upload_egg(target, egg, project, version):
File "/usr/local/bin/scrapyd-deploy", line 195, in _upload_egg
return _http_post(req)
File "/usr/local/bin/scrapyd-deploy", line 211, in _http_post
f = urllib2.urlopen(request)
File "/usr/local/lib/python2.7/urllib2.py", line 154, in urlopen
return opener.open(url, data, timeout)
File "/usr/local/lib/python2.7/urllib2.py", line 431, in open
response = self._open(req, data)
File "/usr/local/lib/python2.7/urllib2.py", line 449, in _open
'_open', req)
File "/usr/local/lib/python2.7/urllib2.py", line 409, in _call_chain
result = func(*args)
File "/usr/local/lib/python2.7/urllib2.py", line 1227, in http_open
return self.do_open(httplib.HTTPConnection, req)
File "/usr/local/lib/python2.7/urllib2.py", line 1200, in do_open
r = h.getresponse(buffering=True)
File "/usr/local/lib/python2.7/httplib.py", line 1073, in getresponse
response.begin()
File "/usr/local/lib/python2.7/httplib.py", line 415, in begin
version, status, reason = self._read_status()
File "/usr/local/lib/python2.7/httplib.py", line 371, in _read_status
line = self.fp.readline(_MAXLINE + 1)
File "/usr/local/lib/python2.7/socket.py", line 476, in readline
data = self._sock.recv(self._rbufsize)
socket.error: [Errno 104] Connection reset by peer